.github/workflows/bump_dev_version_nightly.yml

```diff
@@ -0,0 +1,109 @@
+# This workflow is triggered at 3:00 AM CET everyday and checks if there has
+# been a commit in the last 24 hours on the develop branch of tudat repository.
+
+# If there has been a commit, it bumps the version in the develop branch of
+# the tudat repository, creates a tag with the version numbers and pushes both
+# the commit and the tag to the repository, and then bumps the version in the
+# develop branch of the tudat-feedstock repository and pushes the commit and
+# the tag to the repository.
+
+# This workflow uses personal access tokens to push the changes to the repository. The personal access token is stored as a secret in the repository.
+
+# This workflow file must be in the master (default branch) of the tudat repository because scheduled workflows are only triggered on the default branch.
+
+# Reference for the logic to check if there has been a commit in the last 24 hours: https://stackoverflow.com/questions/4089430/how-can-i-determine-if-a-git-commit-is-more-recent-than-another-commit
+
+
+name: Bump dev version in tudat and tudat-feedstock if new commit in last 24 hours
+
+on:
+    schedule:
+        - cron: '0 1 * * *'  # This triggers the workflow daily at 3:00 AM CET. GitHub uses UTC time, so 1:00 AM UTC is 3:00 AM CET.
```

Comment:
@DominicDirkx please change this line by altering the time at which the workflow is triggered. For example, change the execution time to 2:00 AM UTC (4am NL time)
`cron: '0 2 * * *'`